lib/twemproxy_exporter/counter.rb

```diff
@@ -0,0 +1,19 @@
+module TwemproxyExporter
+  class Counter
+    def initialize(registry, name, desc)
+      @counter = Prometheus::Client::Counter.new(name, desc)
+      registry.register(@counter)
+      @last = 0
+    end
+
+    def count(value, labels = {})
```

Comment:
This function is my biggest open question.  The logic here is to allow the underlying twemproxy counter to be reset (like if it restarts) and keep counting up.  I'm still not sure if this is the correct response.

Curious if you have any thoughts on this.  Do you know how Prometheus handles it if an exporter reboots?